config/schemas/config.yaml

```diff
@@ -14,6 +14,220 @@ $defs:
     thing:
       $ref: "#/$defs/thing"
     ```
+  affinity:
```

Comment:
```suggestion
  kubernetesAffinity:
```
I'd prefer if we prefix any external spec with the source of that spec.

Reply:
Sensible.